src/Commands/Integration.ts

```diff
@@ -184,7 +180,7 @@ export class Integration implements CommandModule {
       await updateConnectivity();
     } catch (e) {
       await notify(ConnectivityStatus.DISCONNECTED);
-      onError(e);
+      onError(new Error(Helpers.getErrorMessage(e)));
```

Comment:
ditto

Reply:
Because param of error in the catch has unknown type.
Helpers.getErrorMessage helps us to cover often used construction for get error message.